src/frontend/src/pages/AdminToolsPage/ProjectsConfig/WorkPackageTemplateTable.tsx

```diff
@@ -16,27 +20,52 @@ const WorkPackageTemplateTable = () => {
     error: workPackageTemplatesError
   } = useAllWorkPackageTemplates();
 
+  const [templateToDelete, setTemplateToDelete] = useState<WorkPackageTemplate>();
+
+  const { mutateAsync } = useDeleteWorkPackageTemplate();
+
   if (!workPackageTemplates || workPackageTemplatesIsLoading) return <LoadingIndicator />;
   if (workPackageTemplatesIsError) return <ErrorPage message={workPackageTemplatesError.message} />;
 
-  const workPackageTemplateRows = workPackageTemplates.map((workPackageTemplateId) => (
+  const workPackageTemplateRows = workPackageTemplates.map((workPackageTemplate) => (
     <TableRow>
       <TableCell align="left" sx={{ border: '2px solid black' }}>
-        {workPackageTemplateId.templateName}
+        {workPackageTemplate.templateName}
       </TableCell>
-      <TableCell sx={{ border: '2px solid black', verticalAlign: 'middle' }}>
-        {workPackageTemplateId.templateNotes}
+      <TableCell sx={{ border: '2px solid black', verticalAlign: 'middle' }}>{workPackageTemplate.templateNotes}</TableCell>
+      <TableCell align="center" sx={{ border: '2px solid black', verticalAlign: 'middle' }}>
+        <IconButton onClick={() => setTemplateToDelete(workPackageTemplate)}>
+          <Delete />
+        </IconButton>
       </TableCell>
     </TableRow>
   ));
 
   return (
-    <Box>
-      <AdminToolTable columns={[{ name: 'Name' }, { name: 'Description' }]} rows={workPackageTemplateRows} />
-      <Box sx={{ display: 'flex', justifyContent: 'right', marginTop: '10px' }}>
-        {isAdmin(currentUser.role) && <NERButton variant="contained">New Work Package Template</NERButton>}
+    <>
+      <Box>
+        <AdminToolTable columns={[{ name: 'Name' }, { name: 'Description' }, { name: '' }]} rows={workPackageTemplateRows} />
+        <Box sx={{ display: 'flex', justifyContent: 'right', marginTop: '10px' }}>
+          {isAdmin(currentUser.role) && <NERButton variant="contained">New Work Package Template</NERButton>}
+        </Box>
       </Box>
-    </Box>
+      <NERModal
+        open={!!templateToDelete}
+        title="Warning!"
+        onHide={() => setTemplateToDelete(undefined)}
+        submitText="Delete"
+        onSubmit={() => {
+          mutateAsync(templateToDelete!.workPackageTemplateId);
+          setTemplateToDelete(undefined);
+        }}
+      >
+        <Typography>
+          Are you sure you want to delete the work package template <i>{templateToDelete?.templateName}</i>?
+        </Typography>
+        <Typography>This will also delete all templates blocked by this one.</Typography>
```

Comment:
Add a statement saying if you wish to only delete this work package template, remove any reference to this template from any other templates blocked bys. But put that into chatgpt to be more clear